diff -urB mysql-5.1.34/sql-bench/test-ATIS mysql-5.1.34-mods/sql-bench/test-ATIS
--- mysql-5.1.34/sql-bench/test-ATIS	2009-05-08 10:31:56.000000000 +0200
+++ mysql-5.1.34-mods/sql-bench/test-ATIS	2009-05-08 10:26:44.000000000 +0200
@@ -38,9 +38,8 @@
   $opt_loop_count/=10;
 }
 
-print "ATIS table test (disabled)\n\n";
+print "ATIS table test\n\n";
 
-=for the rest is disabled
 ####
 ####  Connect and start timeing
 ####
@@ -565,5 +564,3 @@
 # Alias used in joins
   @extra_names=("airport as from_airport","airport as to_airport");
 }
-
-=cut
